src/import.ml

```diff
@@ -162,5 +174,31 @@ module Range = struct
     let end_ = Range.end_ t |> Position.json_of_t in
     Jsonoo.Encode.(object_ [ ("start", start); ("end", end_) ])
 
+  type range =
+    { start : Position.position
+    ; end_ : Position.position
+    }
+
+  let range_of_t t =
+    let start = Range.start t |> Position.position_of_t in
+    let end_ = Range.end_ t |> Position.position_of_t in
+    { start; end_ }
+
+  let t_of_range { start; end_ } =
+    let start = Position.t_of_position start in
+    let end_ = Position.t_of_position end_ in
+    Range.makePositions ~start ~end_
+
   include Vscode.Range
 end
+
+module Node = struct
+  let set_timeout milliseconds =
+    Promise.make (fun ~resolve ~reject:_ ->
+        let (_ : Node.Timeout.t) =
+          Node.setTimeout (fun () -> resolve ()) milliseconds
+        in
+        ())
```

Comment:
I would move this to the actual `node.ml` file with a name that references the fact that it returns a promise. Something like `setTimeoutPromise`.